docs/en/upgrade.md

```diff
@@ -55,7 +55,7 @@ You can view the status of the stack in the AWS CloudFormation console in the **
 
 You can view the status of the pipeline in the solution console in the **Status** column. After a few minutes, you can receive an Active status.
 
-## Post-Upgrade Actions (only applicable to data modeling enabled)
+## Post-Upgrade Actions I (only applicable to data modeling enabled)
```

Comment:
It'd better for adding modules as sub-header, like,

```markdown
### Ingestion Module
...

### Data Modeling

### Upgrade the Data Schema and Out-of-the-box Dashboards

...
```

Reply:
Refactored